NDB.Covid19/NDB.Covid19/PersistedData/LocalPreferencesHelper.cs

```diff
@@ -118,5 +118,11 @@ public static DateTime LastPermissionsNotificationDateTimeUtc
             get => _preferences.Get(PreferencesKeys.LAST_PERMISSIONS_NOTIFICATION_DATE_TIME, DateTime.MinValue);
             set => _preferences.Set(PreferencesKeys.LAST_PERMISSIONS_NOTIFICATION_DATE_TIME, value);
         }
+        
+        public static bool AreCountryConsentsGiven
+        {
+            get => _preferences.Get(PreferencesKeys.COUNTRY_CONSENTS_GIVEN, false);
+            set => _preferences.Set(PreferencesKeys.COUNTRY_CONSENTS_GIVEN, value);
+        }
```

Comment:
Shouldn't it be? 
get........ value
set...... false

Reply:
no, in Get we specify the default value. in Set we specify the value passed to the method